render/detailed_node_test.go

```diff
@@ -109,6 +111,6 @@ func TestMakeDetailedNode(t *testing.T) {
 		},
 	}
 	if !reflect.DeepEqual(want, have) {
-		t.Errorf("%s", diff(want, have))
+		t.Errorf("%s", test.Diff(want, have))
```

Comment:
Personally, not a big fan of `test` as a package name. Maybe, `diff.New(want, have)`??